CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+### Version 0.4.1
+- Merge pull request #74 from teleporthq/hotfix/0.4
```

Comment:
Is there a way to not show the merges in here and actually show the relevant commit messages?

Reply:
Yeah, I will look into that 👍

Reply:
Hi @alexnm  currently ``bump`` doesn't support the omission of merge commits out of the box but I created a [PR](https://github.com/fabiospampinato/bump/pull/5) on their Repo, let's see 😄 

Here is how the changelog will look if that is implemented. 

~~~
### Version 0.4.1
- refactor(resolver): uniform implementation of util functions
- refactor(deprecation): deprecate react-all generator
- fix code samples
- Update README.md
- refactor(dependencies): move extract dependencies inside assembly line
- fix(vue output): remove style tag if it is not needed
- refactor(plugins): using a generic traversal method
- refactor(style plugins): handling dynamic values in nested styles (origin/master, origin/HEAD, master)
- chore(bump.json): Added bump.json for generating changelogs and releases
~~~

Reply:
that looks awesome, I think we can integrate it like this for now, if we can get this feature into `bump` at a certain point in the future